### PR TITLE
Add Checking methods with tests

### PR DESCRIPTION
**Pull request by MoatazMohamedAllam**

### CODE REVIEW

**Comment by josuave** [comment]
I wonder if a better naming may be "Verification"

**Comment by MoatazMohamedAllam** [comment]
yeah you are right it is a better name

**Comment by josuave** [comment]
Is there a reason that these extension methods are in different namespaces?

File: src/Ardalis.Extensions/Checking/IsNull.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Ardalis.Extensions.Checking
+{
+    public static partial class CheckingExtensionMethods
```
Comment (ardalis) [inline src/Ardalis.Extensions/Checking/IsNull.cs:7]:
```suggestion
    public static partial class CheckingExtensions
```

File: src/Ardalis.Extensions/Checking/IsTrue.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Ardalis.Extensions.Checking
+{
+    public static partial class CheckingExtensionMethods
```
Comment (ardalis) [inline src/Ardalis.Extensions/Checking/IsTrue.cs:7]:
```suggestion
    public static partial class CheckingExtensions
```

File: src/Ardalis.Extensions/Verification/IsNull.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Ardalis.Extensions.Verification
+{
+    public static partial class CheckingExtensionMethods
```
Comment (ardalis) [inline src/Ardalis.Extensions/Verification/IsNull.cs:7]:
```suggestion
    public static partial class CheckingExtensions
```

File: src/Ardalis.Extensions/Verification/IsTrue.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Ardalis.Extensions.Verification
+{
+    public static partial class CheckingExtensionMethods
```
Comment (ardalis) [inline src/Ardalis.Extensions/Verification/IsTrue.cs:7]:
```suggestion
    public static partial class CheckingExtensions
```

File: src/Ardalis.Extensions/Verification/IsTrue.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Ardalis.Extensions.Verification
```
Comment (ardalis) [inline src/Ardalis.Extensions/Verification/IsTrue.cs:5]: What is the difference between `Verification` and `Checking` ?

File: tests/Ardalis.Extensions.UnitTests/Verification/IsTrueTests.cs [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+        public void ReturnsTrueGivenTrueCondition(bool condition)
+        {
+            var result = condition.IsTrue();
+
+            Assert.True(result);
+        }
+
+        [Theory]
+        [InlineData(1 == 2)]
+        public void ReturnsFalseGivenFalseCondition(bool condition)
```
Comment (ardalis) [inline tests/Ardalis.Extensions.UnitTests/Verification/IsTrueTests.cs:19]: Probably can be a `[Fact]` not a theory. Same for above.

**Review by ardalis — CHANGES_REQUESTED** [review]

**Pull request closed without merging**